### PR TITLE
Nominate Krsna as the productivity WG lead

### PR DESCRIPTION
**Pull request by chizhg**
As discussed during April 7th TOC meeting.

/cc @dprotaso @kvmware

### CODE REVIEW

File: working-groups/WORKING-GROUPS.md [hunk 1]
```diff
@@ -53,17 +53,22 @@ other system (Apple Calendar or Outlook, for example),
...
-- [Documentation + User Experience](#documentation--user-experience)
-- [Eventing](#eventing)
-- [Eventing Kafka](#eventing-kafka)
-- [Eventing Sources](#eventing-sources)
-- [Networking](#networking)
-- [Operations](#operations)
-- [Productivity](#productivity)
-- [Scaling](#scaling)
-- [Security](#security)
+- [Working Groups](#working-groups)
```
Comment (chizhg) [inline working-groups/WORKING-GROUPS.md:56]: My vscode automatically formatted this.. By checking https://github.com/knative/community/blob/b13ee779c01bad5b30e40444e4ec87f74ee7a9a4/working-groups/WORKING-GROUPS.md it seems to be a desirable change though.

**Comment by dprotaso** [comment]
You'll need to re-base due to: https://github.com/knative/community/pull/1008

/assign @knative/technical-oversight-committee

File: peribolos/knative-sandbox.yaml [hunk 2]
```diff
@@ -770,6 +770,7 @@ orgs:
         members:
         - cardil
         - chaodaiG
+        - chizhg
```
Comment (evankanderson) [inline peribolos/knative-sandbox.yaml:773]: Just an FYI -- sub-teams are automatically members of the parent team, so you shouldn't need this. It doesn't hurt, though.

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**